spec/send_sms_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'send_sms'
+
+describe SendSMS do
+  describe '#send' do
+    it 'sends a sms' do
+      expect(subject).to receive(:send)
+      subject.send
+    end
+  end
+end
```

Comment:
RSPEC runs perfectly and this is a well written test so I assume your texting function is working!